grafana/grafana
# set unit for CloudWatch GetMetricStatistics result (Fixes #13575)

# About this PR
This is a new feature of CloudWatch datasource.
Automatically set graph yaxis unit by using GetMetricStatistics API response.
GetMetricData (math expression) doesn't response unit, so it is not supported.

# How to test
Display the metrics with Unit in Grafana, and check unit is automatically set.
For example, when displaying EC2 `NetworkIn` metrics, the unit should be Bytes. 
https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ec2-metricscollected.html
